include/sharg/argument_parser.hpp

```diff
@@ -91,6 +91,18 @@ namespace sharg
  *
  * ```
  *
+ * ###  The POSIX conventions
+ *
+ * The sharg::argument_parser follows the
+ * [POSIX conventions](https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html). Note that this means
+ * among others:
+ *
+ * 1. Options without arguments can use one hyphen, for example `-a -b` is equivalent to `-ab`.
+ * 2. Between an option and an argument is no space required, for example `-c foo` is equivalent to `-cfoo`.
+ * 3. `--` terminates the options and signals that only position option arguments follow (therefore enabling the user to
+ * use an argument containg an `-` without it being misinterpreted as an option identifier).
+ *
+ *
```

Comment:
I think, you are right, at least I don't see where we would have implemented this (Idk how to check it on the cmd???). Should we open an issue to implement it or mark this as exception?

Reply:
cmdline would be just using `-` whenever you would use an input/output file. Then it should read/write from stdin.
A workaround for output is `--outfile /dev/stdout` since you can use system pipes in our I/O.

> Should we open an issue to implement it or mark this as exception?

Sounds good, I think we should do both. Mention here that the `-` thing is an exception and open an issue, so we don't forget about it.

Reply:
If I change the input in the example of the tutorial like this "parser.add_option(args.file_path,'i', "input", "Please provide a tab separated data file.");", then I can not just use a hyphen for the input, so I would say this is not POSIX conform.

Reply:
Is this ok? https://github.com/seqan/sharg-parser/issues/97